kmath-core/src/commonMain/kotlin/kscience/kmath/expressions/DifferentiableExpression.kt

```diff
@@ -0,0 +1,21 @@
+package kscience.kmath.expressions
+
+/**
+ * And object that could be differentiated
+ */
+public interface Differentiable<T> {
```

Comment:
1. SAM interface.
2. Does it have any uses, except the one in DifferentiableExpression? This trait can be extracted to separate into separate interface at any moment.

Reply:
Removed